components/camel-jetty-common/src/main/java/org/apache/camel/component/jetty/JettyHttpComponent.java

```diff
@@ -1319,6 +1329,8 @@ protected Server createServer() {
                 qtp.setMaxThreads(maxThreads.intValue());
             }
             tp = qtp;
+            _queuedThreadPool=qtp;
```

Comment:
it is a little bit confusing, shouldn't we use `_queuedThreadPool` instead of `qtp` directly from the beginning of `protected Server createServer()` method?

Reply:
done